src/js/components/Index.js

```diff
@@ -98,6 +98,17 @@ export default class Index extends Component {
     }
     const ViewComponent = VIEW_COMPONENT[view];
 
+    let instructional;
+    if (this.props.instructional) {
+      instructional = (
+        <Box className={`${CLASS_ROOT}__instructional`}
+          pad={{horizontal: 'large', vertical: 'small'}}
```

Comment:
i believe the padding should be in the caller's end. I would not even use the Box here.

Also, please see my recommendation with the scss manipulation. I don't think you need a className here. please use Responsive utils to not render the node instead of using display:none;

Can you rename the variable to `preamble` ? I don't really like instructional.